ui/1.5/Dockerfile

```diff
@@ -0,0 +1,58 @@
+####
+# This Dockerfile is used in order to build a container with Debezium UI.
+# It is derived from standard Quarkus-build Docker file but the build is
+# executed from the sources.
+###
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1 AS builder
+
+ARG JAVA_PACKAGE=java-11-openjdk-devel
+ARG BRANCH=master
+
+ENV LANG='en_US.UTF-8' \
+    LANGUAGE='en_US:en' \
+    JAVA_HOME='/usr/lib/jvm/java-11-openjdk'
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} git maven \
+    && microdnf update \
+    && microdnf clean all
+
+RUN java -version \
+    && mkdir -p /sources \
+    && cd /sources \
+    && git clone -b $BRANCH https://github.com/debezium/debezium-ui.git . \
+    && mvn -am dependency:go-offline \
+    && mvn clean package -DskipTests -DskipITs -Dquarkus.package.type=fast-jar
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+
+ARG JAVA_PACKAGE=java-11-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.8
+
+ENV LANG='en_US.UTF-8' \
+    LANGUAGE='en_US:en' \
+    JAVA_HOME='/usr/lib/jvm/jre-11'
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
+    && microdnf update \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown 1001 /deployments \
+    && chmod "g+rwX" /deployments \
+    && chown 1001:root /deployments \
+    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 540 /deployments/run-java.sh \
+    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
+
+# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+
+COPY --from=builder --chown=1001 /sources/backend/target/quarkus-app /deployments/
```

Comment:
For better caching of independent layers Quarkus once updated the default Dockerfile to have 4 layers here. Does it make sense to do this here as well? Compare to the Dockerfile in the UI repo: https://github.com/debezium/debezium-ui/blob/master/backend/src/main/docker/Dockerfile#L58#L61

Reply:
@rk3rn3r I intentionally removed it as I think it is not important for Docker Hub deployed and built images

Reply:
Hum, the lib caching is pretty good, no? I'd like that better, agreeing with @rk3rn3r here.

Reply:
Done

Reply:
@jpechane, it's the same timeout now we see on GH Actions. I suppose the right Maven options need to be added?

Reply:
From my experiences at my last job that timeout issue could be a docker-in-docker issue? Is travis using Docker?
Maybe we can switch to buildah or something that doesn't use docker deamon for building? On kubernetes such service containers often have low specs which lead to sometimes slow and unreliable builds. At least that was the case for us.